scripts/bin/populate-activitypub-db

```diff
@@ -0,0 +1,177 @@
+    }
+}'
+
+curl -X POST http://fake-mastodon:8080/__admin/mappings \
+   -H "Content-Type: application/json" \
+   -d '{
+    "request": {
+        "method": "GET",
+        "urlPattern": "/user/.*"
+    },
+    "response": {
+        "status": 200,
+        "jsonBody": {
+            "@context": [
+                "https://www.w3.org/ns/activitystreams",
+                "https://w3id.org/security/v1",
+                "https://w3id.org/security/data-integrity/v1",
+                "https://www.w3.org/ns/did/v1",
+                "https://w3id.org/security/multikey/v1",
+                {
+                    "manuallyApprovesFollowers": "as:manuallyApprovesFollowers",
+                    "toot": "http://joinmastodon.org/ns#",
+                    "featured": {
+                        "@id": "toot:featured",
+                        "@type": "@id"
+                    },
+                    "featuredTags": {
+                        "@id": "toot:featuredTags",
+                        "@type": "@id"
+                    },
+                    "discoverable": "toot:discoverable",
+                    "suspended": "toot:suspended",
+                    "memorial": "toot:memorial",
+                    "indexable": "toot:indexable",
+                    "schema": "http://schema.org#",
+                    "PropertyValue": "schema:PropertyValue",
+                    "value": "schema:value"
+                }
+            ],
+            "id": "http://fake-mastodon:8080/user/{{request.pathSegments.[1]}}",
+            "type": "Person",
+            "inbox": "http://fake-mastodon:8080/inbox/{{request.pathSegments.[1]}}",
+            "publicKey": {
+                "id": "http://fake-mastodon:8080/user/{{request.pathSegments.[1]}}#main-key",
+                "type": "CryptographicKey",
+                "owner": "http://fake-mastodon:8080/user/{{request.pathSegments.[1]}}",
+                "publicKeyPem": '"$PUBLIC_PEM_JSON"'
+            },
+            "followers": "http://fake-mastodon:8080/followers/{{request.pathSegments.[1]}}",
+            "following": "http://fake-mastodon:8080/following/{{request.pathSegments.[1]}}",
+            "icon": {
+                "type": "Image",
+                "url": "https://ghost.org/favicon.ico"
+            },
+            "liked": "http://fake-mastodon:8080/liked/{{request.pathSegments.[1]}}",
+            "name": "User {{request.pathSegments.[1]}}",
+            "outbox": "http://fake-mastodon:8080/outbox/{{request.pathSegments.[1]}}",
+            "preferredUsername": "{{request.pathSegments.[1]}}",
+            "summary": "This is a dynamically generated summary for {{request.pathSegments.[1]}}",
+            "url": "http://fake-mastodon:8080/user/{{request.pathSegments.[1]}}"
+        },
+        "headers": {
+            "Content-Type": "application/activity+json"
+        }
+    }
+}'
+
+# Define variables
+PRIVATE_KEY="private.pem"
+
+# Loop 5000 times
+for i in $(seq 1 5000); do
```

Comment:
Should we make it so the `5000` can be passed into the script?

Reply:
Yessss